code/modules/transport/pods/vehicle.dm

```diff
@@ -112,6 +113,13 @@
 	attackby(obj/item/W, mob/living/user)
 		user.lastattacked = src
 		if (health < maxhealth && isweldingtool(W))
+#ifdef MAP_OVERRIDE_NADIR
+			var/turf/T = get_turf(src)
+			if(istype(T,/turf/space/fluid) || istype(T,/turf/simulated/floor/plating/airless/asteroid))
```

Comment:
Hypothetically, but because of dynamically allocated areas when tiles are placed, there isn't really a better way to do this detection that I can think of besides the much more lengthy additional process of polling for a present fluid pool, checking not only its depth level but also probably its proportion of aqua tenebrae.

If you think this is likely to be powergamed and it's worth the additional process, I would be willing to give that a whirl, but I don't know that it would be.

Reply:
Well it looks like it worked like that originally, but was changed due to concerns about affecting oshan repair, could just revert that and check whether the active_liquid is actually acid?

Reply:
Did the thing.